### PR TITLE
register screen without connecting DB

### PR DESCRIPTION
**Pull request by ShirAmit96**

### CODE REVIEW

**Comment by ArnonEilat** [comment]
You should not commit the `node_modules` folder.

https://stackoverflow.com/questions/18128863/should-node-modules-folder-be-included-in-the-git-repository

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,25 +1,24 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import './index.css';
+import K_Health_logo from "../src/logoK.svg";
+import img_button from "../src/calanderIcon.svg";
```
Comment (ArnonEilat) [inline frontend/src/App.js:4]:
```suggestion
import img_button from "./calanderIcon.svg";
```

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,25 +1,24 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import './index.css';
+import K_Health_logo from "../src/logoK.svg";
```
Comment (ArnonEilat) [inline frontend/src/App.js:3]:
```suggestion
import K_Health_logo from "./logoK.svg";
```

**Comment by ArnonEilat** [comment]
There are a lot of redundant image files - you can remove them and keep the repo clean. 

Use prettier plug in to format the code on save
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

Better to use vector files rather than raster files. you have the K logo as a vector.

**Review by katz-tom — APPROVED** [review]

**Pull request merged**